_dev/src/components/product-feed/settings/summary/summary.vue

```diff
@@ -249,8 +249,13 @@ export default {
     };
   },
   computed: {
-    disabledExportButton() {
-      return !(this.acceptSyncSchedule && this.understandTerms);
+    disabledExportButton: {
+      get() {
+        return !(this.acceptSyncSchedule && this.understandTerms);
+      },
+      set(value) {
+        return value;
+      },
```

Comment:
There is a better way to fix the issue here. If you want to manage a variable inside your component, it should be defined in the `data()`, not as a computed.

Reply:
If I put it in the data() then it's not dynamically updated when the other data variables are modified

Reply:
Then this computed should depend on another variable you will create, something like `loading` or `apiCallInProgress` from my understanding.

Reply:
It is depending on the state of 2 checkboxes. I don't understand your point with loading/apiCall